src/cli/commands/uploadFile.ts

```diff
@@ -20,7 +20,7 @@ import { UploadFileOptions } from '../types.js';
 import { turboFromOptions } from '../utils.js';
 
 export async function uploadFile(options: UploadFileOptions): Promise<void> {
```

Comment:
TODO: get balance, use approvals first if available, add `--ignore-approvals` or similar to disable this